### PR TITLE
[BZ2054073]: Change the description

### PR DESCRIPTION
**Pull request by xenolinux**
OCP version: 4.6+
[Preview](https://deploy-preview-41902--osdocs.netlify.app/openshift-enterprise/latest/rest_api/network_apis/route-route-openshift-io-v1.html)
QE contact @lihongan-LGTMed 
[Bug](https://bugzilla.redhat.com/show_bug.cgi?id=2054073)

### CODE REVIEW

File: rest_api/network_apis/route-route-openshift-io-v1.adoc [hunk 1]
```diff
@@ -222,7 +221,7 @@ Required::
 | `string`
 | insecureEdgeTerminationPolicy indicates the desired behavior for insecure connections to a route. While each router may make its own decisions on which ports to expose, this is normally port 80.
 
-* Allow - traffic is sent to the server on the insecure port (default) * Disable - no traffic is allowed on the insecure port. * Redirect - clients are redirected to the secure port.
+* Allow - allows insecure connections for an edge-terminated route. * None - disables insecure connections for an edge-terminated route. * Redirect - redirects insecure connections for an edge-terminated route.
```
Comment (nalhadef) [inline rest_api/network_apis/route-route-openshift-io-v1.adoc:224]: Shouldn't this be a bulleted list?

Comment (xenolinux) [reply rest_api/network_apis/route-route-openshift-io-v1.adoc:224]: I am afraid if I use bulleted lists for this column the other content wouldn't be parallel? This column could stand out?

**Comment by lihongan** [comment]
LGTM.
Thank you @xenolinux

File: rest_api/network_apis/route-route-openshift-io-v1.adoc [hunk 2]
```diff
@@ -44,7 +44,6 @@ Required::
 
 | `metadata`
 | xref:../objects/index.adoc#objectmeta-meta-v1[`ObjectMeta meta/v1`]
-|
```
Comment (mburke5678) [inline rest_api/network_apis/route-route-openshift-io-v1.adoc:47]: @xenolinux Did you intend to delete this line? The table doesn't look right in your preview.

File: rest_api/network_apis/route-route-openshift-io-v1.adoc [hunk 3]
```diff
@@ -568,7 +567,6 @@ Defaults to unset
 | Parameter | Type | Description
 | `body`
 | xref:../objects/index.adoc#deleteoptions-meta-v1[`DeleteOptions meta/v1`]
-|
```
Comment (mburke5678) [inline rest_api/network_apis/route-route-openshift-io-v1.adoc:571]: @xenolinux Did you intend to remove this row?

**Comment by mburke5678** [comment]
@xenolinux It seems you removed this column from all of the Body parameters tables. The tables are not rendering correctly in the preview. If you want to eliminate this column in the tables, I think you need to change the header and the first row: `[cols="1,1",options="header"]`
`| Parameter | Type`

File: rest_api/network_apis/route-route-openshift-io-v1.adoc (hunk 1 above)
Comment (xenolinux) [reply rest_api/network_apis/route-route-openshift-io-v1.adoc:224]: okay. Bullets are not rendering in the preview/table. so reverting back

File: rest_api/network_apis/route-route-openshift-io-v1.adoc (hunk 2 above)
Comment (xenolinux) [reply rest_api/network_apis/route-route-openshift-io-v1.adoc:47]: Addressed

File: rest_api/network_apis/route-route-openshift-io-v1.adoc (hunk 3 above)
Comment (xenolinux) [reply rest_api/network_apis/route-route-openshift-io-v1.adoc:571]: No. Added new lines

**Comment by xenolinux** [comment]
> @xenolinux It seems you removed this column from all of the Body parameters tables. The tables are not rendering correctly in the preview. If you want to eliminate this column in the tables, I think you need to change the header and the first row: `[cols="1,1",options="header"]` `| Parameter | Type`

Addressed. Kept `[cols=` as it is since reverted the changes

**Review by lpettyjo — COMMENTED** [review]
LGTM

File: rest_api/network_apis/route-route-openshift-io-v1.adoc (hunk 1 above)
Comment (ctauchen) [reply rest_api/network_apis/route-route-openshift-io-v1.adoc:227]:
```suggestion
* Allow - allows insecure connections for an edge-terminated route. * None - disables insecure connections for an edge-terminated route. * Redirect - redirects insecure connections for an edge-terminated route.
```

If you add the `a` macro before the cell marker `|`, the contents of the cell are interpreted as regular asciidoc format. You can get your unordered list this way.

**Pull request merged**